stdlib/LinearAlgebra/docs/src/index.md

```diff
@@ -354,7 +354,8 @@ LinearAlgebra.tril!
 LinearAlgebra.diagind
 LinearAlgebra.diag
 LinearAlgebra.diagm
-LinearAlgebra.scale!
+LinearAlgebra.mul1!
```

Comment:
Perhaps these should be moved to the `## Low-level matrix operations`-section with `mul!`, `ldiv!` and `rdiv!` further down (~L433).